t/op/hook/require.t

```diff
@@ -0,0 +1,215 @@
+#!perl
```

Comment:
I hadn't noticed the extra directory level here - this test isn't run by `make test_harness`, I searched the first three CI runs that actually run tests and none ran this file.

I added a bare `die;` to this file and all tests passed.

Reply:
Fixed in https://github.com/Perl/perl5/pull/20937